src/Chainweb/Miner/Pact.hs

```diff
@@ -165,7 +166,7 @@ readRewards v =
       let
         !n = v ^. chainGraph . to (int . size)
         !m = fromRational $ toRational b
-      in (BlockHeight a, ParsedDecimal $ m / n)
+      in (BlockHeight a, ParsedDecimal $ roundTo 8 (m / n))
```

Comment:
`roundTo` or `floorTo` ?

Reply:
We agreed that `truncate` was what we wanted - we can use `roundTo' truncate 8 (m / n)` and get this behavior

Reply:
It's all pretty arbitrary for rewards ... also, we picked 8 places just to save space vs 12.